client/components/Calendar.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+
+const Calendar = (props) => {
+  //store month names for retrieval based on an index
+  let months = ['Januray', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
```

Comment:
I think this should be const because it won't change.

Also, are you using the linter because it catches that.